test/numerical/TestConv.cpp

```diff
@@ -61,21 +58,50 @@ int main(int argc, char *argv[]) {
       argc, argv, "TestConv\n", nullptr, "TEST_ARGS");
   std::cout << "Target options: \""
             << getCompilerOption(OptionKind::TargetAccel) << "\"\n";
-
-  // Had to explicitly iterate over dynamic as otherwise the random algorithm
+  // Get configurations from an environment variable
+  std::map<std::string, std::string> opts =
+      ModelLibBuilder::getTestConfigFromEnv("TEST_CONFIG");
+  // Set configuration for test
+  int dimType, maxDilation;
```

Comment:
The code would be more compact if you set `dimType` and `maxDilation` to their default value here, and then only redefine them when an `opts` override them. I find this style easier to read.

Reply:
Done. Thanks!